packages/elements/src/components/ino-chip/ino-chip.scss

```diff
@@ -15,6 +15,18 @@ ino-chip {
   .mdc-chip {
     @include chips.fill-color-accessible($default-chip-color);
 
+    .mdc-chip__icon--trailing {
```

Comment:
Why is this necessary? Do the leading and trailing icons not have the same size?

Reply:
Removing the class results in trailing icons being smaller than leading icons. We're also using the 'mdc-chip__icon--leading' class for the leading icon. However, due to the absence of a corresponding class for trailing icons in the mdc package, I added the 'mdc-chip__icon--trailing' class manually. I also attached a screenshot to illustrate the effect of removing the aforementioned.
![Skjermbilde 2021-02-03 kl  10 50 49](https://user-images.githubusercontent.com/44554211/106730700-f02d2d80-660e-11eb-942c-0eb8e416cf8b.png)

Reply:
Alright, thanks for the explanation ✔️